connection.go

```diff
@@ -751,12 +756,14 @@ func (scd *snowflakeArrowStreamChunkDownloader) GetBatches() (out []ArrowStreamB
 			Loc:     loc,
 			scd:     scd,
 		}
+		logger.Debugf("batch %v, numrows: %v", i, toFill[i].numrows)
 		totalCounted += int64(scd.ChunkMetas[i].RowCount)
 	}
 
 	if len(rowSetBytes) > 0 {
 		// if we had a first batch, fill in the numrows
 		out[0].numrows = scd.Total - totalCounted
+		logger.Debugf("first batch, numrows: %v", out[0].numrows)
```

Comment:
I feel like this log combined with the one in the loop above is a bit confusing, with "first batch" being the last one logged and distinct from "batch 0". Can we move them around a bit or at least change the message to be clearer?

Reply:
Hmmm, first batch is a standard name in all drivers to describe content received with original response. This one is just special. I believe we can leave it as is.